src/legacy/state/orders/utils.ts

```diff
@@ -375,3 +375,6 @@ export function partialOrderUpdate({ chainId, order }: UpdateOrderParams, dispat
   }
   dispatch(updateOrder(params))
 }
+
+// Checks if the order is expired or not
+export const isNotExpired = (order: Order) => Number(order.openSince) + order.validTo > Date.now()
```

Comment:
`openSince` is possibly undefined.
What happens here when it is?

Reply:
Also, here you are adding 2 timestamps, probably not what you want

Reply:
I've changed approach to fix this a bit.
Good point about openSince, I've added a check for that.
I do want to add 2 timestamps, because as far as I can see the openSince is timestamp and validTo is just duration of time that the order is valid expressed in miliseconds. So to get expiry date we transform openSince to ms and add validTo.

Reply:
I'm still unsure regarding `validTo`, because it is stored as a timestamp AFAICT, no?

Looking at the docs string from cow-sdk: https://github.com/cowprotocol/cow-sdk/blob/a622134dbdd10a3340724b9ae259f034561ae562/src/order-book/generated/models/OrderParameters.ts#L42

That's why I don't think adding 2 timestamps will work.

Unless one of them here is a relative time delta rather than a timestamp, in which case the description/name on them is confusing...